examples/step-38/doc/intro.dox

```diff
@@ -118,7 +118,7 @@ This surface description given, in the codimension one surface case, the two
 routines FEValues::shape_grad and FEValues::JxW
 return
 @f{align*}
-\text{FEValues::shape\_grad}(i,l)&=D \mathbf x_K(p_l) G^{-1}(p_l)D(\varphi_i \circ \mathbf x_K)
+\text{FEValues::shape_grad}(i,l)&=D \mathbf x_K(p_l) G^{-1}(p_l)D(\varphi_i \circ \mathbf x_K)
```

Comment:
Normally, one needs to write "\_" in Latex. But indeed in the documentation it is not rendered correctly. Does anyone have an idea for the cause?

Reply:
Maybe it really is the `\text` command, that treats the `_` symbol as an underscore rather than the subscript command and thus does not have to be escaped. I found that in deal.II, we need to escape the underscore within the `\mathrm` command.

Well, if @peterrum's suggestions doesn't work, we can still try out `\textunderscore`.

Reply:
I believe that it's because we don't actually run our formulas through latex, but have doxygen translate it into MathJax. But it would be a shame to lose the ability to use latex, which we probably do when generating the offline documentation, or at least might want to preserve the ability to keep doing.

My suggestion would be to just rewrite the whole text as follows:
```
  The surface description given, in the codimension one face, the two routines we need are the following:
  - FEValues::shape_grad(), which returns $...$
  - FEValues::JxW(), which returns $...$
```
where the text in `$...$` is simply a copy of the right hand sides of the formulas already used.

Reply:
Thanks @bangerth, I took your suggestion (hope I understood correctly) and I added a commit to this pr.